code/controllers/subsystems/characters/database_persistence.dm

```diff
@@ -185,7 +185,7 @@ SUBSYSTEM_DEF(database)
 	unknown_designs = list()
 
 	//This gets a list of every design that exists
-	var/list/designs = SSresearch.design_ids.Copy()
+	var/list/designs = SSresearch.design_by_id.Copy()
```

Comment:
this seems an unnecessary change
at least make it "designs_by_id"

Reply:
"desigsn_by_id" makes more sense. Var was renamed because of three reasons: "design_ids" doesn't describe actuall purpose, "design_by_id" was var that existed in `/datum/research`, I had to check if all cases where it is required use it in right way.